### ISSUE TITLE
remotelink capability?

### ISSUE THREAD

**Issue by ctramnitz** [lesstif/php-jira-rest-client#67]
Would it be possible to add an option to update a ticket with a "remotelink" as described [here](https://developer.atlassian.com/jiradev/jira-platform/guides/other/guide-jira-remote-issue-links/jira-rest-api-for-remote-issue-links#JIRARESTAPIforRemoteIssueLinks-CreatingLinks)?

**Comment by lesstif** [comment]
The feature is't currently yet available.

thanks!

**Comment by Programie** [comment]
This feature is still not available yet, right?

Maybe this will help: https://developer.atlassian.com/server/jira/platform/jira-rest-api-for-remote-issue-links/

I used another JIRA client before which let me send custom HTTP requests to the JIRA API. As far as I can see, this library allows to send custom requests using the `exec()` method of the IssueService.

So this might work:

```php
$issueService->exec(sprintf("issue/%s/remotelink", $issue->key), json_encode(array(
    "object" => array(
        "url" => $url,
        "title" => $title
    )
)), "POST");
```

`$issue` is an instance of the Issue class (e.g. returned by the `create()` method of the IssueService).

Maybe just add that as an additional method for the IssueService just like the other ones like `addWatcher()`?

**Comment by lesstif** [comment]
hi @Programie @ctramnitz 

this feature implemented at Ver 1.19.

see the [manual](https://github.com/lesstif/php-jira-rest-client#remote-issue-link)

thanks.
